rest-api-spec/src/main/resources/rest-api-spec/test/cluster.allocation_explain/10_basic.yaml

```diff
@@ -0,0 +1,75 @@
+---
+"cluster shard allocation explanation test":
+  - do:
+      # there aren't any unassigned shards to explain
+      catch: request
+      cluster.allocation_explain: {}
+
```

Comment:
I'd probably add `---` here and make a test case. These are difficult to debug so I take any opportunity I can to split them into smaller and smaller chunks. Totally up to you though.